package.json

```diff
@@ -44,7 +44,7 @@
     "mini-css-extract-plugin": "1.3.0",
     "node-sass": "4.13.1",
     "optimize-css-assets-webpack-plugin": "5.0.3",
-    "postcss-loader": "3.0.0",
+    "postcss-loader": "4.0.4",
```

Comment:
Seemingly nothing that impacts us - we were already on required webpack/node versions.  Out usage of a postcss config file also means we don't need to worry about the `postcssOptions` loader change:

https://github.com/webpack-contrib/postcss-loader/blob/master/CHANGELOG.md#400-2020-09-07